### PR TITLE
updated the create a new test case page

### PR DESCRIPTION
**Pull request by ma-zah**
pls review, thanks!

### CODE REVIEW

**Comment by sker65** [comment]
lgtm

**Review by fabiomaienschein — APPROVED** [review]

File: new-test-case.mdx [hunk 1]
```diff
@@ -1,44 +1,70 @@
...
+    alt="A generated step failed and shows a yellow alert"
   />
 </Frame>
 
-### Chaining your tests
+### Pre-prompted login test
+
+If your app has a login functionality that wasn't AI discovered at the set-up, we have a pre-prompted login test case for you. You can trigger it after giving us test user credentials.
+
+Go to `settings` a.k.a. the `gear icon` in the upper right corner and select `test account`. Add test user credentials we can use to run the login test.
```
Comment (fabianboth) [inline new-test-case.mdx:44]: Do we want to mention that this is also created on the fly for a new project, if we detect a login test case and the user provides credentials there?

**Review by fabianboth — APPROVED** [review]
mini nitpick

File: new-test-case.mdx (hunk 1 above)
Comment (ma-zah) [reply new-test-case.mdx:44]: aaaah, good one. I'll do a new commit.

**Pull request merged**